src/common.h

```diff
@@ -34,6 +34,11 @@
 # define LIB32_ARCH               LD_I386_LIB32
 # define USR_LIB_MULTIARCH_DIR    "/usr/lib/x86_64-linux-gnu"
 # define USR_LIB32_MULTIARCH_DIR  "/usr/lib/i386-linux-gnu"
+#elif defined(__powerpc64__)
+# define LIB_ARCH                 LD_POWERPC_LIB64
+# define LIB32_ARCH                 LD_POWERPC_LIB64
+# define USR_LIB_MULTIARCH_DIR    "/usr/lib/powerpc64le-linux-gnu"
+# define USR_LIB32_MULTIARCH_DIR    "/usr/lib/powerpc64le-linux-gnu"
```

Comment:
I would prefer `/var/empty` for the `USR_LIB32_MULTIARCH_DIR`, this would be a little more explicit.
We should probably also have `OPT_COMPAT32 = 0 << 6` set in `options.h` for PPC64LE to enforce it.